cmake/configs/default.cmake

```diff
@@ -99,6 +99,7 @@ hunter_config(convertutf VERSION 1.0.1)
 hunter_config(crashpad VERSION v0.0.1-p0)
 hunter_config(crashup VERSION 0.0.2)
 hunter_config(cvmatio VERSION 1.0.27-p3)
+hunter_config(czmq VERSION 4.0.1-p0)
```

Comment:
Ok, I don't get it, how can i test czmq package, if czmq package is not part of hunter?
Where and how should i set version for CI?